src/writing-your-own-application/alloc.md

```diff
@@ -0,0 +1,53 @@
+# Alloc
+
+In `no_std`, we need to rely on `alloc` crate, which provides heap allocation support. To use `alloc` in `no_std` environment:
```

Comment:
I think we should reword this to change the sentiment. We don't "need" to, but it's there as an option if a user needs to:

* use a crate that needs alloc
* use the well known dynamic collections like `Vec`

Reply:
I updated the wording I hope it's more clear now.